File: leetcode-problems/problem-1672.py
# 1672. Richerst customer wealth 

class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        max_wealth = 0
        for customer_accounts in accounts:
            if sum(customer_accounts) > max_wealth:
                max_wealth = sum(customer_accounts)
        return max_wealth
        

File: leetcode-problems/problem-1672.go
// 1672.

package problem1672

func maximumWealth(accounts [][]int) int {
	wealthiestFortune := 0
	currentFortune := 0
	for i := 0; i < len(accounts); i++ {
		for j := 0; j < len(accounts[i]); j++ {
			currentFortune += accounts[i][j]
		}
		if currentFortune > wealthiestFortune {
			wealthiestFortune = currentFortune
		}
		currentFortune = 0
	}
	return wealthiestFortune
}
